<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1193</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Log In And Registration System</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src.qrc">
    <normaloff>:/images/C:/Users/asus/Downloads/login.jpg</normaloff>:/images/C:/Users/asus/Downloads/login.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralWidget{
 border-image: url(:/images/C:/Users/asus/Downloads/bg.png);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="topPanel" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#topPanel{
   border-image:url(:/images/C:/Users/asus/Downloads/header.jpeg);
}</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>0</y>
       <width>801</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QWidget" name="loginIcon" native="true">
        <property name="styleSheet">
         <string notr="true">#loginIcon{
    border-image:url(:/images/C:/Users/asus/Downloads/login.jpg);
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">#label{

color:white;
}</string>
        </property>
        <property name="text">
         <string>LogIn And Registration System</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="loginForm">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>1151</width>
      <height>541</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#loginForm{

	background-color: rgb(0, 170, 0);
}</string>
    </property>
    <property name="title">
     <string>Login</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>100</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Username:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="userNameLogin">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>90</y>
       <width>471</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordLogin">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>210</y>
       <width>471</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>220</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loginBtn">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>330</y>
       <width>361</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#loginBtn{
   color:white;
   background-color:rgb(0, 0, 255);
   border-radius:10px;
}

#loginBtn:hover { 
  color: rgb(0, 0, 0);
  background-color: rgb(255, 85, 0);

}</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLabel" name="accountLabel">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>410</y>
       <width>361</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#accountLabel{
color:rgb(170, 0, 0)

}</string>
     </property>
     <property name="text">
      <string>Don't Have Acoount? Please </string>
     </property>
    </widget>
    <widget class="QPushButton" name="registerDailougOpen">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>410</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#registerDailougOpen{

   color:white;
   background-color:;
	background-color: rgb(170, 85, 255);
   border-radius:5px;
}

#registerDailougOpen:hover{
  color: rgb(0, 0, 0);
  background-color: rgb(255, 85, 0);
}</string>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
    <widget class="QLabel" name="usernameValidation">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>160</y>
       <width>471</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#usernameValidation{
color:red;

}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="passwordValidation">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>280</y>
       <width>471</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#passwordValidation{
 color:red;

}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
